project.mml

```diff
@@ -1519,6 +1519,8 @@ Layer:
             tags->'tower:type' as "tower:type",
             tags->'castle_type' as castle_type,
             tags->'information' as information,
+            tags->'visibility' as "visibility",
+            tags->'support' as support,
```

Comment:
Why "visibility" is in quotes and support is not?

Reply:
purly esthetical, will add the quotes to follow others